notebooks/xeus-cpp.ipynb

```diff
@@ -499,7 +497,7 @@
     "    nl::json mime_bundle_repr(const image& i)\n",
     "    {\n",
     "        auto bundle = nl::json::object();\n",
-    "        bundle[\"image/png\"] = xtl::base64encode(i.m_buffer.str());\n",
+    "        bundle[\"image/png\"] = xeus::base64encode(i.m_buffer.str());\n",
```

Comment:
Well I thought this would do the job for us :(

Reply:
@anutosh491 Don't you need to load a xeus header to use it?